.github/workflows/cd.yaml

```diff
@@ -0,0 +1,25 @@
+name: CD
+
+on:
+  release:
+    types:
+      - published
+
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Publish AUR package
+        uses: KSXGitHub/github-actions-deploy-aur@v2.2.5
+        with:
+          pkgname: scala-cli
+          pkgbuild: scala-cli/PKGBUILD
+          commit_username: ${{ secrets.AUR_USERNAME }}
+          commit_email: ${{ secrets.AUR_EMAIL }}
+          ssh_private_key: ${{ secrets.AUR_SSH_PRIVATE_KEY }}
```

Comment:
Oh CD only runs where there is a release, the rationale is that we can trigger the publish by manually creating a release in GitHub, quite convenient for package maintainers.
As for CI, it only runs on PRs targeting `main` or commits pushed to `main`. They are mutually exclusive, since running CI again is redundant when we want to publish, they are already tested

Reply:
Just wondering if there is a probability release could be done without the CI not having passed. I think that is why we run CI usually before CD jobs

Reply:
Re-read your question, now I know where you are coming from haha.

There could be situation where CI failed, but we still want to publish anyway, hence maintainer should make the conscious decision here.

Reply:
Maybe we can forgo this human error

Reply:
Yeah it's possible to run CI again and then CD, let me do that.

Reply:
Easier if I merge both workflow.

Reply:
Yeah. That is what I usually do as well in my GitHub action

Reply:
There are other alternatives as well

Reply:
Anyway this is what I usually do: https://github.com/yzia2000/foodie/blob/main/.github/workflows/ci.yml